src/main/java/uk/gov/register/presentation/SignedTreeHead.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.register.presentation;
```

Comment:
I wonder if we might want to take these classes in to a separate package for CT stuff? maybe u.g.r.proofs.certificate-transparency?